cloudify_hostpool/rest/backend/acquire.py

```diff
@@ -0,0 +1,71 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+
+
+import uuid
+
+from cloudify_hostpool import exceptions
+from cloudify_hostpool.backend import scan
+
+
+def acquire(db):
+    '''"Creates" (acquires) a host.
+
+    :return: host dict if successful, None otherwise.
+    :param db: host database.
+    '''
+    for host in _aquisition_gen(db):
```

Comment:
i makes it a bit difficult to understand when you use an iteration variable outside of the iteration loop.
i think a better construct would just be returning the host from inside the loop and not using break statements.